src/SimpleGraphs/generators/staticgraphs.jl

```diff
@@ -372,3 +372,44 @@ function CliqueGraph(k::T, n::T) where {T <: Integer}
     add_edge!(g, 1, (n - 1) * k + 1)
     return g
 end
+
+"""
+    LadderGraph(n)
+
+Create a graph consisting of `2n` nodes and `3n-2` edges.
```

Comment:
If I would not know what a ladder graph is, I would still not know after this description. I don't know if that is a good idea, but you could even add an ascii picture such as:
```
o -- o -- o -- ...
|    |    |
o -- o -- o -- ...
```

Reply:
Not in favor of ascii art. Suggest the wikipedia article as a reference: https://en.wikipedia.org/wiki/Ladder_graph

Reply:
What about saying a `square grid of two rows and n columns`?